### DISCUSSION TITLE
Negative Temperatur als große positive Zahl in MQTT

### DISCUSSION THREAD

**Discussion by FWallenwein** [tbnobody/OpenDTU#445] [Q&A] [answered]
Heute Nacht war es das erste mal wirklich kalt, seit ich openDTU laufen 
habe.
Eine negative Temperatur wird im MQTT als große positive Zahl 
übertragen.

Habt Ihr das auch, oder ist meine openDTU Version zu alt ?

Ansonsten läuft openDTU nun seit Monaten klaglos und stabil hier durch.
Super gut.

Gruss Frank

**Comment by tbnobody** [comment, marked as answer]
Siehe #432 und #246
Das Problem wurde am 20. Oktober behoben....

**Reply by FWallenwein** [reply]
Danke. Dann muss ich jetzt doch mal updaten. 
Die alte Version lief gut und stabil, deshalb habe ich erst mal keinen Grund zum update gehabt.
